modules/persistent-storage-local-create-cr.adoc

```diff
@@ -50,7 +50,7 @@ spec:
       volumeMode: Filesystem <3>
       fsType: xfs <4>
       devicePaths: <5>
-        - /path/to/device
+        - /path/to/device <6>
```

Comment:
Hi @qinpingli - Thanks for your comments. I reviewed Furuta-san's BZ comment (https://bugzilla.redhat.com/show_bug.cgi?id=1804489#c4) and now have a better understanding of what he is requesting. Therefore, I've removed the machine config procedure that was originally a part of this PR, which you were understandably confused by. :)

Could you please re-review my changes against his BZ comment and see if this now makes sense? Is the example path that I provided (pulled from the `$ oc get all -n local-storage` output below correct? Or should I use a different example name, such as `/dev/xvdg` as it was there previously?

Thanks!

Reply:
Add an explanation to this field lgtm.